app/components/contact_details/contact_details.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# app/components/contact_details/contact_details_component.rb
+class ContactDetails < ViewComponent::Base
+  # rubocop:disable Metrics/ParameterLists
+  def initialize(name: nil, phone: nil, url: nil, email: nil, facebook_link: nil, twitter_handle: nil, instagram_handle: nil, is_valid_phone: nil)
```

Comment:
this function signiture was very naive of me back in 2017 when i was still learning rails - i think lets use the opp to refactor this to just take the @partner as the function input?

(may also be as places used to be a sep model)